crowbar_framework/lib/crowbar/deployment_queue.rb

```diff
@@ -197,25 +193,21 @@ def process_queue
 
     private
 
-    def commit_proposals(list)
-      list.map do |item|
-        logger.debug("process queue: item to do: #{item.inspect}")
-        bc = item[:barclamp]
-        inst = item[:inst]
+    def commit_proposal(bc, inst)
+      logger.debug("process queue: committing item: #{bc}:#{inst}")
 
-        service = eval("#{bc.camelize}Service.new logger")
+      service = eval("#{bc.camelize}Service.new logger")
```

Comment:
Sure; but as you can see it was not introduced in this PR. Do you want to submit a PR for this? Or shall I do it?